newclient/src/apollo/exceptions/errorExceptions.ts

```diff
@@ -2,6 +2,11 @@
 /* LIST OF ALL API ERRORS TO BE HANDLED ON FRONT END AS NONE ERRORS */
 /*------------------------------------------------------------------*/
 
+/**
+ *  @event -  This error is received, when there is an invalid hash. Happens when a user is trying to search for a valid hash or address.
+ **/
+const excpInvalidHash = 'Invalid Hash'
```

Comment:
you have to make it lower case.  since in isAPIExceptionProduction you are checking against  newE = errorMessage.toLocaleLowerCase()